web-components/src/components/input/Input.ts

```diff
@@ -292,8 +299,19 @@ export namespace Input {
         }
         event.preventDefault();
       }
+      this.input.click();
+      this.value = '';
+      this.dispatchEvent(
+        new CustomEvent("input-change", {
```

Comment:
looks a duplicate code , handleChange doing same thing right

Reply:
updated the function to handle all conditions